.travis.yml

```diff
@@ -23,10 +22,8 @@ matrix:
           python: "2.6"
         - env: DJANGO='Django==1.8.*'
           python: "2.6"
-        - env: DJANGO='Django==1.3.*'
-          python: "3.4"
-        - env: DJANGO='Django==1.4.*'
-          python: "3.4"
+        - env: DJANGO='Django==1.9.*'
+          python: "2.6"
```

Comment:
I was diagonal reading assuming just the opposite (run only in python 2.6)
You're right there's nothing to change. 

Kudos!